qa/tasks/ceph_manager.py

```diff
@@ -1455,7 +1457,8 @@ def run_ceph_w(self, watch_channel=None):
         if watch_channel is not None:
             args.append("--watch-channel")
             args.append(watch_channel)
-        return self.controller.run(args=args, wait=False, stdout=StringIO(), stdin=run.PIPE)
+        return self.controller.run(args=args, wait=False, stdout=StringIO(),
+                                   stdin=run.PIPE, shell=shell)
```

Comment:
````
2021-03-09T07:03:23.289 INFO:tasks.cephfs_test_runner:======================================================================
2021-03-09T07:03:23.289 INFO:tasks.cephfs_test_runner:ERROR: test_backtrace_repair (tasks.cephfs.test_forward_scrub.TestForwardScrub)
2021-03-09T07:03:23.290 INFO:tasks.cephfs_test_runner:----------------------------------------------------------------------
2021-03-09T07:03:23.290 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2021-03-09T07:03:23.290 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_0c43faad14f2a9e42833aa3e6d5a8dcf2da14e66/qa/tasks/cephfs/test_forward_scrub.py", line 294, in test_backtrace_repair
2021-03-09T07:03:23.291 INFO:tasks.cephfs_test_runner:    with self.assert_cluster_log("bad backtrace on inode"):
2021-03-09T07:03:23.291 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_0c43faad14f2a9e42833aa3e6d5a8dcf2da14e66/qa/tasks/ceph_test_case.py", line 112, in __enter__
2021-03-09T07:03:23.292 INFO:tasks.cephfs_test_runner:    shell=False)
2021-03-09T07:03:23.292 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_0c43faad14f2a9e42833aa3e6d5a8dcf2da14e66/qa/tasks/ceph_manager.py", line 1461, in run_ceph_w
2021-03-09T07:03:23.292 INFO:tasks.cephfs_test_runner:    stdin=run.PIPE, shell=shell)
2021-03-09T07:03:23.293 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_git_teuthology_3082387bba74fcd24c9700593d10418152d53c97/teuthology/orchestra/remote.py", line 215, in run
2021-03-09T07:03:23.293 INFO:tasks.cephfs_test_runner:    r = self._runner(client=self.ssh, name=self.shortname, **kwargs)
2021-03-09T07:03:23.293 INFO:tasks.cephfs_test_runner:TypeError: run() got an unexpected keyword argument 'shell'
````

From: /ceph/teuthology-archive/pdonnell-2021-03-09_03:27:39-fs-wip-pdonnell-testing-20210308.214827-distro-basic-smithi/5948385/teuthology.log

Reply:
Ok, this brings a small upgrade to my understanding. Thanks, anyways!